core/domain/subtopic_page_domain_test.py

```diff
@@ -333,25 +322,27 @@ def test_to_and_from_dict(self):
         }
         subtopic_page_contents = (
             subtopic_page_domain.SubtopicPageContents.from_dict(
-                subtopic_page_contents_dict))
+                subtopic_page_contents_dict)) # type: ignore[arg-type]
```

Comment:
Why is this ignore needed?

Reply:
Similar reasons as tha above [ignore](https://github.com/oppia/oppia/pull/14955#discussion_r816918727).